neurogym/config/base.py

```diff
@@ -10,13 +10,27 @@
 
 
 class ConfigBase(BaseSettings):
-    """Configuration base class."""
+    """Base class for project configuration models.
 
-    # This is a built-in attribute that is overriden here.
+    Extends Pydantic's `BaseSettings` to support loading from environment variables
+    and other sources, with custom behavior controlled via `model_config`.
+
+    Key `model_config` options:
+    - `env_prefix`: Only environment variables with this prefix (e.g., "NGYM_") are used.
+    - `env_nested_delimiter`: Enables nested fields (e.g., "NGYM_DB.USER").
```

Comment:
This should probably be `NGYM_DB__USER`. Should we give a real example, like `NGYM_ENV__NAME`?